README.md

```diff
@@ -10,8 +14,57 @@
 
 
 ## Usage
-* Tests must end in `_(spec|test).(coffee|js)`
-* Run tests with ```mycha run```
+
+* `mycha install <testHelperPath>`
+  * runs
+    ```
+    npm install --save-dev chai
+    npm install --save-dev sinon
+    npm install --save-dev sinon-chai
+    ```
+
+  * writes to `<testHelperPath>`
+    ```coffee
+    chai = require 'chai'
+    sinon = require 'sinon'
+    chai.use require 'sinon-chai'
+
+    global.expect = chai.expect
+    global.sinon = sinon
+
+    process.env.NODE_ENV = 'test'
+    ```
+
+  * writes to `mycha.coffee`
```

Comment:
I personally like coffee files. 
The only difference in the file is I need to add `modules.exports`
The difference is use is that I can require the file instead of needing another module to parse it.